### DISCUSSION TITLE
Unable to connect one Search Results webpart to two Search Boxes on separate pages

### DISCUSSION THREAD

**Discussion by Sephiroth0327** [microsoft-search/pnp-modern-search#3756] [General]
**Version used**
4.11.1

**Describe the bug**
I currently have a PnP - Search Box on my home page and my PnP - Results webpart on a separate page.  I have this configured properly and results are filtered as expected.  But I want to place another PnP - Search Box on the same page as the Results as often times users do a search from the home page and would like to do additional searches without needing to keep navigating back to the homepage.  The 2nd search box is correctly updating the URL with the correct hashtag reference but the results do not filter.  I have to actually hit Enter in the Address bar to get it to filter again.  Is this a bug or do I have this misconfigured?

**To Reproduce**
Detailed steps to reproduce the behavior:
See above

**Expected behavior**
See above

**Desktop (please complete the following information):**
 - Desktop - Chrome browser Version 124.0.6367.119

**Comment by kasperbolarsen** [comment]
Your description is not enough info, can your share this setting from your Results web part?
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/d8245190-6eba-41a1-b422-92d554cd3163)

**Comment by Sephiroth0327** [comment]
> Your description is not enough info, can your share this setting from your Results web part? ![image](https://private-user-images.githubusercontent.com/20593570/331665093-d8245190-6eba-41a1-b422-92d554cd3163.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTU5NjI5ODUsIm5iZiI6MTcxNTk2MjY4NSwicGF0aCI6Ii8yMDU5MzU3MC8zMzE2NjUwOTMtZDgyNDUxOTAtNmViYS00MWExLWI0MjItOTJkNTU0Y2QzMTYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTE3VDE2MTgwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTIwOGQ3NTI3NGJkYTFiMGMxZTgyZmFkOGJmYzcyZjIxOWU4ZDk3NDUwOGU3MDU5NzU2MjRjZmZhOTdkZTc4OWUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Kv39CQ431lrWeT5wjjY1OQgWFUhm5oayl6iwA52ofwc)

![image](https://github.com/microsoft-search/pnp-modern-search/assets/69170603/a153740e-35cd-42ee-8f66-9a3b57c33233)

Here you go

**Comment by kasperbolarsen** [comment]
I think I have got it to work the way you want it

The search box is grabbing the initial query from the URL and passing it on to the Results web part. After that you can type any query in the search box and it will update the Results web part.

![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/b2e3fe4f-0b71-47c2-8d7d-a1f0399737ca)


![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/45ba0c5b-f1e7-40a3-a030-24c215d8b953)
